x-pack/plugins/security_solution/public/cases/containers/configure/use_configure.test.tsx

```diff
@@ -165,6 +173,61 @@ describe('useConfigure', () => {
     });
   });
 
+  test('sort of happy save case configuration - getCaseConfigure', async () => {
+    // When there is no version, a configuration is created. Otherwise is updated.
+    const spyOnGetCaseConfigure = jest.spyOn(api, 'getCaseConfigure');
+    spyOnGetCaseConfigure.mockImplementation(() =>
+      Promise.resolve({
+        ...caseConfigurationCamelCaseResponseMock,
+        error: 'uh oh homeboy',
+        version: '',
+      })
+    );
+
+    await act(async () => {
+      const { waitForNextUpdate } = renderHook<string, ReturnUseCaseConfigure>(() =>
+        useCaseConfigure()
+      );
+      await waitForNextUpdate();
+      await waitForNextUpdate();
+      expect(mockErrorToToaster).toHaveBeenCalled();
+    });
+  });
+
+  test('sort of happy save case configuration - postCaseConfigure', async () => {
```

Comment:
nit -> I think a better message would help a future developer to understand the purpose of the test